gradle-maven-push.gradle

```diff
@@ -213,7 +213,8 @@ afterEvaluate { project ->
         if (project.getPlugins().hasPlugin('com.android.application') ||
                 project.getPlugins().hasPlugin('com.android.library')) {
             archives androidSourcesJar
-            archives androidJavadocsJar
+            // todo AdellHk - javadocs classpath is broken eg. BaseDeepLinkDelegate.java:8: error: package androidx.core.app does not exist
```

Comment:
I would not merge with this change - Javadoc is important and we should figure out why it isn't working.

A good fix may be to migrate away from the antiquated gradle script, and use a plugin like https://github.com/vanniktech/gradle-maven-publish-plugin

Reply:
I also think you might not be able to publish to Sonatyope with missing Javadoc.

I think the problem here is not the publishing, but the actual class path for the JavaDoc.